### DISCUSSION TITLE
The default glossary

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#5391] [Ideas]
Having a an overall project for all projects (like the explanation of a meta category from [ref. project categorization](https://github.com/WeblateOrg/weblate/discussions/5388)) allows there to be a overall glossary for it too. This means most entries don't have to be replicated between projects, but rather special terms can go in the respective project glossaries. Since glossaries are components now, matching translations can be imported from there.

With the way things scale, the bulk of strings are bound to be shorter, and that takes a lot of effort away from translating a new project.

It also makes the most out of effort spent, and helps educate translators somewhat. Currently this scales really badly to the capacity of one person.

Like launchpad, seeing where a translation is used makes it a lot easier to ensure consistency.
![bilde](https://user-images.githubusercontent.com/13802408/107474094-0c325100-6b72-11eb-87c0-eaa3c75894ed.png)
If it is a true one-to-many relation, even figuring out how often terms are used gets easier.

The same pinpointing could be done for terms that aren't supposed to be used.
(Edit: Haven't used https://docs.weblate.org/en/latest/user/glossary.html?highlight=forbidden#forbidden-translations yet.)

In more of an ideal world, it would be great to have a dictionary for every language consisting of every word that exists on Weblate for the language. (Federation between instances with venn-diagrams of word-use would be great.) Then with a per-reviewer system things can be vetted and marked either as good, neutral, or bad. Bad entries would trigger checks during translations, and doing neutral ones would show good ones as suggestions, ala Pootle.

**Comment by nijel** [comment]
I have no clue how that works on Norwegian, but for Czech, we have many disputes between various localization teams on how some strings should be translated. There is small shared glossary (https://wiki.l10n.cz/P%C5%99ekladatelsk%C3%BD_slovn%C3%ADk), but  many strings diverge.
